src/Sql.fs

```diff
@@ -504,6 +504,7 @@ module Sql =
         for (paramName, value) in row do
             
             let normalizedParameterName =
+                let paramName = paramName.TrimEnd()
```

Comment:
Looks good! Though why not simply `paramName.Trim()` so that the whitespace gets trimmed both ways :smile:?

Reply:
@Zaid-Ajaj that's a very good point!